### ISSUE TITLE
clang-10 fails to build Carla-2.1: cannot assign to variable 'newIndex' with const-qualified type 'const size_t' (aka 'const unsigned long')

### ISSUE THREAD

**Issue by yurivict** [falkTX/Carla#1150]
```
Compiling CarlaPluginSFZero.cpp
In file included from CarlaPluginSFZero.cpp:22:
In file included from ../../modules/sfzero/SFZero.h:23:
In file included from ../../modules/sfzero/sfzero/SFZSound.h:13:
In file included from ../../modules/water/synthesisers/Synthesiser.h:30:
./../modules/water/synthesisers/../containers/OwnedArray.h:725:30: error: cannot assign to variable 'newIndex' with const-qualified type 'const size_t' (aka 'const unsigned long')
                    newIndex = numUsed - 1;
                    ~~~~~~~~ ^
./../modules/water/synthesisers/../containers/OwnedArray.h:718:56: note: variable 'newIndex' declared const here
    void move (const size_t currentIndex, const size_t newIndex) noexcept
                                          ~~~~~~~~~~~~~^~~~~~~~
1 error generated.
```

Log: http://beefy18.nyi.freebsd.org/data/head-amd64-default/p536541_s361492/logs/Carla-2.1.log (IPv4 URL)

OS: FreeBSD 12.1

**Comment by falkTX** [comment]
Fixed in dcd400f94c3325ef5e796f8029ab051bb0fec8be

**Comment by falkTX** [comment]
FYI 2.1.1 will be out hopefully next week, bugfix only release. will contain this one.
